packages/frontend/public/loader/boot.js

```
@@ -7,13 +7,21 @@
 (async () => {
+	// renderError 自身が throw すると、それが unhandledrejection を再発火させて
+	// onunhandledrejection ハンドラから renderError が再呼び出しされる無限ループに陥り、
+	// メインスレッドを焼き尽くしてエラー画面が操作不能になる。
+	// それを防ぐための再入ガード。
+	let renderErrorRunning = false;
+
 	window.onerror = (e) => {
 		console.error(e);
 		renderError('SOMETHING_HAPPENED', e);
 	};
 	window.onunhandledrejection = (e) => {
```

ESLint

[error] 'window' is not defined.

(no-undef)